dist/timer.js

```diff
@@ -0,0 +1,25 @@
+const TIMEOUT_MS = 60 * 100;
+
+export class Timer {
```

Comment:
I think we do not need this wrapper around `setTimeout`

Reply:
Okey, I will delete it.